bmp3.h

```diff
@@ -46,13 +46,777 @@
 #define BMP3_H_
 
 /* Header includes */
-#include "bmp3_defs.h"
+//#include "bmp3_defs.h"
+
+/*! @file bmp3_defs.h
+ * @brief Sensor driver for BMP3 sensor */
+
+/*!
+ * @defgroup BMP3 SENSOR API
+ * @brief
+ * @{*/
 
 /*! CPP guard */
 #ifdef __cplusplus
 extern "C" {
 #endif
 
+#include <stdint.h>
+#include <stddef.h>
+
+/*************************** Common macros   *****************************/
+

```

Comment:
Below references Linux kernel header files, I adapted it to a non-Linux kernel environment.